cirq/pasqal/pasqal_device.py

```diff
@@ -11,52 +11,45 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-from typing import Any, Dict, FrozenSet, Iterable
-from numpy import sqrt
+from typing import FrozenSet, Sequence
 
 import cirq
-
-from cirq.pasqal import ThreeDGridQubit
+from cirq.ops import NamedQubit
 
 
 @cirq.value.value_equality
 class PasqalDevice(cirq.devices.Device):
-    """A Pasqal Device with qubits placed on a 3D grid."""
+    """A generic Pasqal device."""
```

Comment:
A bit more documentation here would be very helpful (you can just rephrase a bunch of the text you had in the documentation) for users to understand the intent of this class: When is this useful? Why would someone want to use an unconstrained Pasqal device vs the 3D one?